docs/logging/docs/001-overview-logging.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Overview
+type: Overview
+---
+
+Logging in Kyma uses [Logspout](https://github.com/gliderlabs/logspout) and [OK Log](https://github.com/oklog/oklog). Logspout is a log router for Docker containers that runs inside Docker. It attaches to all containers on a host, then routes their logs to a Log Management System. OK Log is a distributed and free from co-ordination log management system. It is a lightweight solution which is not only easy to understand but also easy to operate.
```

Comment:
...It attaches to each container on a host and routes their logs to a Log Management System. OK Log is a distributed coordination-free log management system......

Reply:
done